### PR TITLE
feat: add joke api

### LINKED ISSUE #124: Jokes API
Leverage https://github.com/taivop/joke-dataset for a jokes api

### PR DESCRIPTION
**Pull request by h1z3y3**
close #124

### CODE REVIEW

File: joke/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Joke Service
```
Comment (asim) [inline joke/README.md:1]: Always add 1 line above ``# [X] Service` that acts as a one line descriptor for cards displayed on m3o.com

```
Funny jokes

# Jokes Service

...
```

Comment (h1z3y3) [reply joke/README.md:1]: Got it

**Comment by asim** [comment]
So I've merged it and its running, but I would like to see if we can filter out offensive jokes or find a way to let people flag them by id so they can be blocked/removed. How does that sound?

**Comment by asim** [comment]
It also looks like category is blank when reddit is the source, so maybe we could just set category to Reddit in that case

**Comment by asim** [comment]
Either that or we should ideally remove reddit jokes, as they appear highly offensive

**Comment by asim** [comment]
I won't host on the site until we do that

**Comment by h1z3y3** [comment]
Haha, not as a native English speaker, I actually don't understand them.

But I'll remove Reddit source as you said

**Comment by asim** [comment]
I think we need a better data set. This is pretty poor.

**Comment by asim** [comment]
This has clean jokes from reddit https://github.com/amoudgl/short-jokes-dataset/blob/master/data/reddit-cleanjokes.csv

**Pull request merged**